### ISSUE TITLE
cannot connect，0.44.0

### ISSUE THREAD

**Issue by secondwifelikemytalent** [cars10/elasticvue#147]
The ES I started in the k8s cluster exposed the nodeport port. My browser page can access the node information, but the Elasticvue plugin in the same edge browser shows that it cannot connect, my version is 0.44.0, I can directly click the address in the unable to connect warning below to jump to the ES cluster "You Know, for Search".

<img width="1175" alt="Snipaste_2023-06-16_09-15-14" src="https://github.com/cars10/elasticvue/assets/45771250/b158d0dc-e683-4a56-b997-27dfc6ed156e">

<img width="730" alt="Snipaste_2023-06-16_09-15-49" src="https://github.com/cars10/elasticvue/assets/45771250/c07c3e9c-5b5d-46ef-8e0c-2148682d1a16">

**Comment by lwxk** [comment]
cannot connect same, and miss history connect data

**Comment by cars10** [comment]
Hey @secondwifelikemytalent , can you please provide the following information?

**Environment (please include the following information):**
* Elasticsearch version:
* Operating system:
* Browser + version:
* Elasticvue version:
* How are you running elasticvue? (docker, browser extension, web app):

**Comment by cars10** [comment]
Not reproducible, reopen if you have more details on the issue

**Comment by nattfalk** [comment]
I have the exact same problem with Elasticvue v0.44.0 but I think it might be related to the webbrowser and forcing http to https. I have a local instance of Elastic running in docker on http://localhost:9200. Up until a couple of weeks ago (I think) everything worked fine. 
The problem occurs in both Chrome (version 117.0.5938.62) and Edge (version 116.0.1938.81) and gives the exact same error as above.
![image](https://github.com/cars10/elasticvue/assets/12427715/88876e4c-7f2e-4b09-9f96-1883c64cc84f)

However, if I run Firefox (version 117.0.1) with the extension then it works and connects to the Elastic instance without problem.
![image](https://github.com/cars10/elasticvue/assets/12427715/509a753b-0e44-4a3c-8f60-46ddd12fe31f)

**Comment by cars10** [comment]
> The problem occurs in both Chrome (version 117.0.5938.62) and Edge (version 116.0.1938.81) and gives the exact same error as above.

How do you use elasticvue in these browsers? Browser extension, web version, docker, ..

**Comment by nattfalk** [comment]
I ran Elasticvue as an extension in all browsers under Windows 11 against an Elastic instance running in Docker.

But I found a solution to the problem in Chrome by removing the security policies that force http to https for localhost.
Apparently Firefox does not enforce these strict https rules yet.

Go to chrome://net-internals/#hsts in Chrome and enter "localhost" (without quotationmarks) under **Delete domain security policies** and press Delete and then run Elasticvue extension again. Now it should work.
![image](https://github.com/cars10/elasticvue/assets/12427715/401966b2-2b0b-4a01-add0-3ce3390ab0fa)
![image](https://github.com/cars10/elasticvue/assets/12427715/75308b1e-db37-437a-bd71-e4b7ddf318a8)
